include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -72,17 +73,16 @@ def TT_BitcastOp : TT_Op<"bitcast", [SameOperandsAndResultShape,
     // TODO: Add verifier
 }
 
-def TT_FpToFp : TT_Op<"fp_to_fp", [SameOperandsAndResultShape,
-                                   SameOperandsAndResultEncoding,
-                                   NoSideEffect,
-                                   /*DeclareOpInterfaceMethods<CastOpInterface>*/]> {
+def TT_FpToFpOp : TT_Op<"fp_to_fp", [SameOperandsAndResultShape,
+                                     SameOperandsAndResultEncoding,
+                                     NoSideEffect,
+                                     DeclareOpInterfaceMethods<CastOpInterface>]> {
     let summary = "Floating point casting for custom types";
 
     let description = [{
-        Floating point casting for custom types (F8, BF8).
+        Floating point casting for custom types (F8).
```

Comment:
What is the original background of the "BF8" type? And what is the motivation for removing it in this PR?

Reply:
There are no docs/related information about the BF8 format on the original master branch or on the Internet. And it only appears as a definition without implement in the compiler backend. So I just removed it.

Reply:
BF8 is E5M2; FP8 is E4M3. Followed the same conventions as FP16, choosing the prefix BF for the variant with more exponent bits.